launcher/ui/pages/global/LauncherPage.ui

```diff
@@ -38,7 +38,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
```

Comment:
Change this to 0 (this happens when you accidentally stay on another tab and then save the UI in designer. Just open designer again and click the first tab and then save the file.)